CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # Changelog
 
+## ☀️ 0.5.0
+
+- ### ✨ Features
+
+  - #### 🍱 Module Support
+
+    - **BREAKING: use correct `package.json` keys for generated JavaScript - [ashleygwilliams], [issue/309] [pull/312]**
+
+      [issue/309]: https://github.com/rustwasm/wasm-pack/issues/309
+      [pull/312]: https://github.com/rustwasm/wasm-pack/pull/312
+
+  - #### 🛠️ New Commands
+
+    - **`wasm-pack init` is now `wasm-pack build` - [csmoe], [issue/188] [pull/216]**
+
+      [csmoe]: https://github.com/csmoe
+      [issue/188]: https://github.com/rustwasm/wasm-pack/issues/188
+      [pull/216]: https://github.com/rustwasm/wasm-pack/pull/216
+
+    - **add new command: `wasm-pack test` - [fitzgen], [pull/271]**
```

Comment:
Maybe worth calling out that this will manage headless browser tests for you, which sucks to set up IME.

Maybe also link to https://rustwasm.github.io/wasm-bindgen/wasm-bindgen-test/index.html ?